.readthedocs.yaml

```diff
@@ -0,0 +1,14 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
```

Comment:
```suggestion
# Set the version of Python and other tools you might need
build:
  os: ubuntu-22.04
  tools:
    python: "3.9"
```

Reply:
@adam-a-a I'm not sure but I've noticed that RTD is defaulting to Python 3.7, which might be too old for some of your dependencies. I'd try adding that snippet to the RTD config file to see if a newer Python version fixes the problem.